app/src/main/java/com/project/squirrelobserver/squirrelObserver/RecordBehaviorTabActivity.java

```diff
@@ -269,8 +278,49 @@ private void hideButtons(String filter, ArrayList<ToggleButton> buttons) {
 
                 button.setVisibility(View.GONE);
             } else {
```

Comment:
It's more readable if written like this:

} else if (isScanMode || behavior.isAO) {